swap_meet/vendor.py

```diff
@@ -1,2 +1,66 @@
 class Vendor:
-    pass
+    
+    def __init__(self, inventory=None): # Set a default value
+        if inventory is None: # Default value is falsy, so set to empty list
+            self.inventory = []
+        else:
+            self.inventory = inventory # If inventory is given, set it equal to inventory attribute
```

Comment:
Consider the following refactor to enhance readability

```suggestion
        if not inventory is: # Default value is falsy, so set to empty list
             inventory = []
        self.inventory = inventory # If inventory is given, set it equal to inventory attribute
```